lib/elements/date.js

```diff
@@ -27,16 +27,48 @@ const dfltLocales = {
 }
 
 /**
- * DatePrompt Base Element
+ * DatePrompt
+ * Base
+ * Element
  * @param {Object} opts Options
  * @param {String} opts.message Message
- * @param {Number} [opts.initial] Index of default value
- * @param {String} [opts.mask] The format mask
- * @param {object} [opts.locales] The date locales
- * @param {String} [opts.error] The error message shown on invalid value
- * @param {Function} [opts.validate] Function to validate the submitted value
- * @param {Stream} [opts.stdin] The Readable stream to listen to
- * @param {Stream} [opts.stdout] The Writable stream to write readline data to
+ * @param {Number} [opts.initial] Index
```

Comment:
Can you fix the formatting here? Maybe it's a bug on GitHub?

![image](https://user-images.githubusercontent.com/2302254/60399138-3dd9de00-9b2e-11e9-96ec-3ce926eb4a60.png)

Reply:
Fixed! Don't know why it did that. My formatting settings are setted so my IDE should not even touch the comments. That's weird.